src/proto/flwr/proto/exec.proto

```diff
@@ -52,3 +55,8 @@ message ListRunsResponse {
   map<uint64, Run> run_dict = 1;
   string now = 2;
 }
+message StopRunRequest {
+  uint64 run_id = 1;
+  Fab fab = 2;
```

Comment:
What's the FAB here for? `LinkState.update_run_status` doesn't need it, no?

Reply:
I've kept only the `run_id` for now, as discussed in our thread.